src/plugins/intel_gpu/tests/unit/test_cases/col_to_im_gpu_test.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils.h"
+#include "random_generator.hpp"
+
+#include <intel_gpu/primitives/input_layout.hpp>
+#include <intel_gpu/primitives/reorder.hpp>
+#include <intel_gpu/primitives/col_to_im.hpp>
+
+#include <cstddef>
+
+using namespace cldnn;
+using namespace ::tests;
+
+static void test_col_to_im_output(bool is_caching_test) {
+    auto& engine = get_test_engine();
+
+    auto input = engine.allocate_memory({ data_types::f32, format::bfyx, { 1, 12, 9, 1 } });
+    auto output_size = engine.allocate_memory({ data_types::f16, format::bfyx, { 1, 1, 1, 2 } });
+    auto kernel_size = engine.allocate_memory({ data_types::f16, format::bfyx, { 1, 1, 1, 2 } });
+
+    set_values(input, {
+        1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f, 1.0f,
+        2.0f, 2.0f, 2.0f, 2.0f, 2.0f, 2.0f, 2.0f, 2.0f, 2.0f, 2.0f, 2.0f, 2.0f,
+        3.0f, 3.0f, 3.0f, 3.0f, 3.0f, 3.0f, 3.0f, 3.0f, 3.0f, 3.0f, 3.0f, 3.0f,
+        4.0f, 4.0f, 4.0f, 4.0f, 4.0f, 4.0f, 4.0f, 4.0f, 4.0f, 4.0f, 4.0f, 4.0f,
+        5.0f, 5.0f, 5.0f, 5.0f, 5.0f, 5.0f, 5.0f, 5.0f, 5.0f, 5.0f, 5.0f, 5.0f,
+        6.0f, 6.0f, 6.0f, 6.0f, 6.0f, 6.0f, 6.0f, 6.0f, 6.0f, 6.0f, 6.0f, 6.0f,
+        7.0f, 7.0f, 7.0f, 7.0f, 7.0f, 7.0f, 7.0f, 7.0f, 7.0f, 7.0f, 7.0f, 7.0f,
+        8.0f, 8.0f, 8.0f, 8.0f, 8.0f, 8.0f, 8.0f, 8.0f, 8.0f, 8.0f, 8.0f, 8.0f,
+        9.0f, 9.0f, 9.0f, 9.0f, 9.0f, 9.0f, 9.0f, 9.0f, 9.0f, 9.0f, 9.0f, 9.0f,
+    });
+    set_values(output_size, {
+        ov::float16(4.0f), ov::float16(4.0f)
+    });
+    set_values(kernel_size, {
+        ov::float16(2.0f), ov::float16(2.0f)
+    });
+
+    topology topology;
+    topology.add(cldnn::input_layout("input", input->get_layout()));
+    topology.add(cldnn::data("output_size", output_size));
+    topology.add(cldnn::data("kernel_size", kernel_size));
+    topology.add(cldnn::reorder("reorder_input", input_info("input"), cldnn::layout(data_types::f16, format::byxf, { 1, 12, 9, 1 })));
+    topology.add(cldnn::col_to_im("col_to_im", input_info("reorder_input"), input_info("reorder_input"), input_info("reorder_input"),
```

Comment:
in this test, `output_size` and `kernel_size` are not used at all. Does it work correctly?

Reply:
yes. by other parameter. As input[1],[2] are removed, this confusion would be resolved.